NEWS

```diff
@@ -968,6 +968,7 @@ None
     (no arrows or math signs).
   - Corrected a bug in 8 dots grade 2, which resulted in the
     "var" contraction not always being properly applied.
+
```

Comment:
Hm, the changes in the NEWS file seem like white space changes. Can you add a real description of the changes? I guess you could just use a condensed version of the description of the PR